accounts/ManageConsents.postman_collection.json

```diff
@@ -0,0 +1,1174 @@
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "user_id",
+									"value": "{{userId}}",
+									"type": "text"
+								},
+								{
+									"key": "scope",
+									"value": "credentials:read",
+									"type": "text"
+								},
+								{
+									"key": "",
+									"value": "accounts.balances:readonly,accounts:read,credentials:read,balance-refresh,balance-refresh:readonly,credentials:refresh,credentials:write,user:read",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "{{base}}api/v1/oauth/authorization-grant",
+							"host": [
+								"{{base}}api"
+							],
+							"path": [
+								"v1",
+								"oauth",
+								"authorization-grant"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "3. Generate User Token",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);",
+									"pm.environment.set(\"accessToken\", jsonData.access_token);"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{accessToken}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "client_id",
+									"value": "{{client_id}}",
+									"type": "text"
+								},
+								{
+									"key": "client_secret",
+									"value": "{{client_secret}}",
+									"type": "text"
+								},
+								{
+									"key": "grant_type",
+									"value": "authorization_code",
+									"type": "text"
+								},
+								{
+									"key": "code",
+									"value": "{{code}}",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "{{base}}api/v1/oauth/token",
+							"host": [
+								"{{base}}api"
+							],
+							"path": [
+								"v1",
+								"oauth",
+								"token"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "4. Call Credentials List",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);",
+									"pm.environment.set(\"cred_id\", jsonData.credentials[0].id);"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{accessToken}}",
+								"type": "default"
+							},
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"value": "application/x-www-form-urlencoded",
+								"type": "text"
+							},
+							{
+								"key": "report_id",
+								"value": "6b20c4cc062a45fd9ed852402f982523",
+								"type": "text",
```

Comment:
Should the report_id be there?

Reply:
Good catch. It should placeholder and not the actual value. Updated it.